backend/api/Areas/Tools/Helpers/ImportPropertiesHelper.cs

```diff
@@ -253,6 +253,7 @@ private Entity.Parcel AddUpdateParcel(Model.ImportPropertyModel property, int pi
                 p_e.AgencyId = agency?.Id ??
                     throw new InvalidOperationException($"Agency '{property.Agency}' does not exist.");
                 p_e.Agency = agency;
+                p_e.Name = property.Name ?? property.Description?.Substring(0, 150 < property.Description.Length ? 150 : property.Description.Length).Trim();
```

Comment:
If `name` is included it will use the `description` to keep it backward compatible.